.github/workflows/command_shell_acceptance.yml

```diff
@@ -1,5 +1,5 @@
 
-name: Acceptance
+name: Command Shell Acceptance
```

Comment:
Renaming jobs as it was confusing having everything in Github's UI being called 'Acceptance'